salt/modules/virt.py

```diff
@@ -673,6 +634,117 @@ def init(name,
                     )
                     define_vol_xml_str(xml)
 
+            elif hypervisor in ['qemu', 'kvm']:
+
```

Comment:
As I said - we should iterate 'diskp' for any type of hypervisors, the difference is: for qemu-kvm we create image, for other hypervisors we just describe the volume for libvirt (define_vol_xml_str() )